src/map-caption.js

```diff
@@ -0,0 +1,45 @@
+import './leaflet.js'; 
+import './mapml.js'; 
+
+/* 
+implemented for both mapml-viewer and web-map; however web-map does not focus on map element in the browser resulting in NVDA 
+not being able to read out map-caption and stating that it's an interactive map region
+*/
+export class MapCaption extends HTMLElement {
+    constructor() {
+        super();
+    }
+
+    // called when element is inserted into DOM (setup code)
+    connectedCallback() {
+
+        // calls MutationObserver; needed to observe changes to content between <map-caption> tags and update to aria-label
+        let mapcaption = this.parentElement.querySelector('map-caption').innerHTML;
```

Comment:
For mapcaption, I need this to return the first map-caption only so that I can store it to then compare it to the mapcaption that was removed/modified.